### DISCUSSION TITLE
Is it possible to use the floating toolbox with other toolbars?

### DISCUSSION THREAD

**Discussion by nicolae-stroncea** [xournalpp/xournalpp#2868] [Q&A]
I primarily use the "Right Hand Note Taking Toolbar". I would like to add the floating toolbox to my workflow and add some tools there that I can access quickly.

When I map the floating toolbox to a stylus button, it shows up as **Empty Toolbox**(*Menu: View/Toolbars*) unless I change the toolbar to  "Experimental Floating Toolbox" toolbar.

**Comment by rolandlo** [comment]
The problem is, that currently you cannot have any tools duplicated (toolbar and floating toolbox). PR #2735 will change that.
